src/models/permissions.js

```diff
@@ -0,0 +1,20 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+'use strict';
+
+module.exports = (sequelize, DataTypes) => {
+  const Permission = sequelize.define('Permissions', {
+    name: { type: DataTypes.STRING, primaryKey: true }
+  }, {
+    timestamps: false,
+    classMethods: {
+      associate: db => {
+        Permission.belongsToMany(db.Clients, { through: 'ClientPermissions' });
```

Comment:
I just tried using a single association here and it does not work for what we need. We need to be able to assign the same permission to multiple clients, so we need a n:m relation. With a `Client.hasMany(Permissions)` (1:n relation) we get a Permissions table with a Client's foreign key, which is not enough for our needs. For example, with this kind of relation, trying to set the same permission (`admin`) to two different clients ends up with this result:

```json
[{
    "key": "acb91322e69ed852",
    "name": "anotherClientName",
    "authRedirectUrls": [ "http://domain.org" ],
    "authFailureRedirectUrls": [ "http://domain.org" ],
    "permissions": [ "admin" ]
  }, {
    "key": "f8913a3a75deffdf",
    "name": "clientName",
    "authRedirectUrls": ["http://domain.org"],
    "authFailureRedirectUrls": ["http://domain.org"],
    "permissions": []
  }
]
```
As you can see, the permissions list of the second client is empty.